students/arkadiusz_kasprzyk/lesson_06_dicts/uppercase.py

```diff
@@ -0,0 +1,91 @@
+# uppercase.py
+
+
+def capitalize_word(word):
+    """
+    Parameters
+    ----------
+    word: str
+
+    Returns
+    -------
+    word_cap: str
+        First letter of `word` turned to uppercase (if it's a letter).
+
+    Examples
+    --------
+    capitalize_word("spam")         # 'Spam'
+    capitalize_word("Spam")         # 'Spam'
+    capitalize_word("_spam")        # '_spam'
+    capitalize_word(" spam")        # ' spam'
+    capitalize_word("_spam eggs")   # '_spam eggs'
+    """
+    word0 = word[0]
+    char0 = ord(word0)
+    if char0 in range(97, 123):
```

Comment:
You assume the encoding here, when it's actually a Unicode character, not an ASCII codepoint. There is a method `upper()` and even `capitalize()`...

Reply:
Of course there are proper methods, but using them makes the task trivial! What's the point of such excercises!? I did it this way because it was little bit more interesting! (Not a perfect solution but at least sth to think.)

Reply:
Just saying, it's not equivanent. I don't need any fixes here.